src/app/common/base-url.ts

```diff
@@ -1,3 +1,3 @@
-var  baseUrl: string = 'REPLACE ME';
+var  baseUrl: string = 'http://ada.cs.pdx.edu:3333/';
```

Comment:
I'm sure it's safe to have ada.cs.pdx.edu here, but wouldn't it be better in general to keep REPLACE ME here?